backend/routes/verifyPlanInvite.ts

```diff
@@ -0,0 +1,7 @@
+import express from 'express'
+import InviteContactController from '../controllers/inviteContacts';
+const verifyPlanInviteRouter = express.Router()
+
+verifyPlanInviteRouter.get('/:id', InviteContactController.verifyPlanInvite)
```

Comment:
I am thinking your auth logic could go as a middle ware here

Reply:
yup i will put the `checkUserAuthorization` middleware at the router level